getting-started/README.md

```diff
@@ -1,110 +1,108 @@
+## Create and Configure a Virtual Server
+1.  Navigate to [IBM Cloud](https://cloud.ibm.com). - Click the **Create resource** button at the top right of the screen. Select **Compute** from the options available and select `Virtual Server for VPC`
 
-- Software Requirements: Refer to the [Support Matrix](https://github.com/opea-project/GenAIExamples/blob/main/README.md#getting-started) to ensure you have the required software components in place.
+2. Select a location for the instance. Assign a name to it.
 
-Note : If you are deploying it on cloud, say AWS, select a VM instance from R7iz or m7i family of instances with base OS as Ubuntu 22.04 (AWS ami id : ami-05134c8ef96964280). Use the command below to install docker on a clean machine.
-```
-wget https://raw.githubusercontent.com/opea-project/GenAIExamples/refs/heads/main/ChatQnA/docker_compose/install_docker.sh
-chmod +x install_docker.sh
-./install_docker.sh
-```
-## Understanding OPEA's Core Components
+3. Under Stock Images, select Ubuntu 24.04 (`ibm-ubuntu-24-04-6-minimal-amd64-1`)
 
-Before moving forward, it's important to familiarize yourself with two key elements of OPEA: GenAIComps and GenAIExamples.
+4. Select a virtual server.
+> **Note:** We recommend selecting a 3-series instance with an Intel(R) 4th Gen Xeon(C) Scalable Processor, such as `bx3d-16x80` or above. For more information on virtual servers on IBM cloud visit [Intel® solutions on IBM Cloud®](https://www.ibm.com/cloud/intel).
 
-- GenAIComps is a collection of microservice components that form a service-based toolkit. This includes a variety of services such as llm (language learning models), embedding, and reranking, among others.
-- While GenAIComps offers a range of microservices, GenAIExamples provides practical, deployable solutions to help users implement these services effectively. Examples include ChatQnA and DocSum, which leverage the microservices for specific applications.
+5. Create or add an ssh key for the instance.
 
-## Visual Guide to Deployment
-To illustrate, here's a simplified visual guide on deploying a ChatQnA GenAIExample, showcasing how you can set up this solution in just a few steps.
+6. Click on `Create virtual server` at the bottom right.
 
-![Getting started with OPEA](assets/getting_started.gif)
+7. Once the instance is running, create and attach a `Floating IP` to the instance. For more information visit [this](https://cloud.ibm.com/docs/vpc?topic=vpc-fip-working&interface=ui) site
 
-## Setup ChatQnA Parameters
-To deploy ChatQnA services, follow these steps:
+8. `ssh` into the instance using the floating IP (`ssh -i <key> ubuntu@<floating-ip>`)
 
+
+
+## Deploy the ChatQnA Solution
+Use the command below to install docker on a clean virtual machine
+```bash
+wget https://raw.githubusercontent.com/opea-project/GenAIExamples/refs/heads/main/ChatQnA/docker_compose/install_docker.sh

```

Comment:
This script might move ..
Why not a put a copy click thing on page and reproduce the instructions in the install_docker.sh inline?

Reply:
@mkbhanda will add a fair few bit of lines - https://raw.githubusercontent.com/opea-project/GenAIExamples/refs/heads/main/ChatQnA/docker_compose/install_docker.sh

Reply:
Actually no, you have 3 lines here, and downloading docker stuff directly is 1 more line.